### PR TITLE
Square Viewport + Default Separated Mode

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR

A bit of a double whammy.

Primarily, the game viewport is now a 1:1 aspect ratio (15 tiles by 15 tiles.) This is primarily because, due to being able to rotate the camera, player's have an advantage in how much they can see around them based on the orientation of their camera, which is really just a scummy feature to have. With a square viewport, you essentially have the same view of the area around you no matter how much the camera is rotated.

Defaulting to separated mode is really just a reality of the classic mode not working well with non-native aspect ratios. The more square the ratio, the bigger the ugly bars on the side of the screen get. I personally think that having this space filled in by anything, even just chat, is a vastly better visual experience for the player. No need to hide it in the chat.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

omitted the aspect ratio because it's fairly obvious.

:cl:
- tweak: The default chat layout has been changed to the "separated" style. This can be reverted in the settings.

### CODE REVIEW

**Comment by deathride58** [comment]
Strong, *strong*, ***strong*** disagree with squaring the viewport, as there's *far* better solutions to the issue of people rotating their viewport for the visibility (for instance, as I've proposed in another PR: allowing people to bring their mouse to the edges of the screen to pan their view in that direction)

From a UX standpoint, a square viewport means that the overwhelming majority of the screen becomes blank space, and with separated chat specifically, can genuinely cause eyestrain from how off-center the viewport ends up. We have always been a strong advocate for a wider viewport ever since it became feasible in SS13, and we are absolutely *not* a fan of regressing the game in this way.

**Comment by moonheart08** [comment]
I do not like this for the reasons Lank and Bhijn have already stated.

**Comment by Tunguso4ka** [comment]
if it is not gonna be merged as a default, then maybe make it toggleable? Like we did with chat. So you can have your preferred view: standard or squared.

would get abused by the powergamers though.

**Comment by moonheart08** [comment]
> if it is not gonna be merged as a default, then maybe make it toggleable? Like we did with chat. So you can have your preferred view: standard or squared.
> 
> would get abused by the powergamers though.

it already is, there's a slider in options.

**Comment by deltanedas** [comment]
if you could toggle it why would you *not* toggle it

**Comment by Tunguso4ka** [comment]
> it already is, there's a slider in options.

oh, never seen it.
okay then.

**Comment by PJB3005** [comment]
From day 1 of this project I have asserted that the viewport should be centered, exactly for some of the reasons already pointed out here: SS13's viewport is ugly as hell, and having to look to the side of your screen constantly *sucks*. The viewport size was also picked with the intent of getting something that's "not square, but not too rectangular to make rotation a hugely significant mechanic".

> I personally think that having this space filled in by anything, even just chat, is a vastly better visual experience for the player.

It's a ton of visual clutter for a mechanic that I have always wanted to reduce the importance of. The fact of the matter is that a huge chat window should not be necessary for normal gameplay. It *is* in SS13, but SS14 puts vastly different stuff in there so it's not the same.

**Comment by EmoGarbage404** [comment]
Im perfectly fine leaving the aspect ratio as is and removing those modifications from the pr, but i still think separated should be the default style. I honestly don't think that it's a matter of emphasis placed on the chat window.

Between the two options, only one of them has a significant portion of the HUD dedicated to dead space. Is the side of the screen being only chat perfect? Not necessarily, but also, there's way more that could be done with it in the future. Integrating the PDA or a statpanel into the separated mode sidebar could do a lot to make the space more meaningful.

At the very least, it's not being wasted space. The default layout is unnecessarily burdensome in that it's constructed with a ton of small unusable space thats only appeal is sticking small windows into it so that it doesnt cover the game (something you can accomplish equally well with the large chat sidebar.)

I can't really fault criticisms that it's offcenter, but i think it's far from being outright ugly. Its just wasted potential to let screenspace go to waste instead of being able to better integrate it in some way.

**Comment by OtherSkittle** [comment]
> ...and having to look to the side of your screen constantly sucks.

> I can't really fault criticisms that it's offcenter, but i think it's far from being outright ugly.

Has eye dominance been considered in the past when it comes to this? I mention this because I wonder if people thinking that separated mode looks ugly is related to the fact that not only is a left sided viewport off center but it's even more off the center of the right eye (70% of people are right eye dominant and assuming that people center their head with the screen, the right eye would be slightly to the right of the center of the screen). An option to move the chat bar to the left would make it so that the distance between the right eye and the center of the viewport is less than the current version.

**Comment by TurboTrackerss14** [comment]
As a dedicated seperated gamer. 
I am bias and attached to the old ss13 "style" of GUI in total but even I do not support moving seperated to default. 
It's very "aged" composition wise and is honestly just violently jank (resizing the chat box a tiny amount will implode the entire chat scrolling)

I am also a camera rotation "abuser", rotating the camera to see what... 2 ish tiles further? Yeah its simply gross and I pulled back from using it so much..... I used to rotated camera at every corner during chases... 


TLDR; 
Don't make seperated default, square viewport is a bandaid fix to what could be solved in better ways.
